pom.xml

```diff
@@ -1938,19 +1937,6 @@
           </exclusion>
         </exclusions>
       </dependency>
-      <!-- Hive 2.3 need this to init Hive's FunctionRegistry -->
-      <dependency>
-        <groupId>org.codehaus.jackson</groupId>
-        <artifactId>jackson-core-asl</artifactId>
-        <version>${codehaus.jackson.version}</version>
-        <scope>${hive.jackson.scope}</scope>
```

Comment:
Can we also remove `<hive.jackson.scope>compile</hive.jackson.scope>`?
https://github.com/apache/spark/blob/44f00ccebfd2b55862212dc2ac6c0c5b03c86227/pom.xml#L270

Reply:
if we identified some issues on hive 2.3.10 before 4.0.0 release, we may need to revert this patch and fallback to SPARK-47119 approach to mitigate CodeHaus Jackson dependencies vulnerabilities, see comemnts at
https://github.com/apache/spark/pull/45201#issuecomment-2103839853

Reply:
ok

Reply:
Ya, sorry for making things difficult, @pan3793 and @wangyum .

If we are sure, we can clean up more easily definitely.